src/view/MainViewManager.js

```diff
@@ -845,6 +846,35 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Switch between panes
+     */
+    function switchPaneFocus() {
+        var $firstPane = $('#first-pane'), $secondPane = $('#second-pane');
+        if($firstPane.hasClass('active-pane')) {
+            $secondPane.click();
+        }
+        else {
+            $firstPane.click();
+        }
+    }
+
+    /**
+     * Unit test for switching from first pane to second pane
```

Comment:
@arthur801031 exactly. I created an issue for adding the modifier key option https://github.com/adobe/brackets/issues/12859